.github/CHANGELOG.md

```diff
@@ -240,6 +240,7 @@
 1. [RMP] Added integral illumination to RMP volume knobs - @ImenesFBW (Imenes)
 1. [RMP] Fixed ACP CALL buttons emissive behavior - @ImenesFBW (Imenes)
 1. [TEXTURE] Fixed decal clipping issues on the overhead fuel panel - @ImenesFBW (Imenes)
+1. [ECAM] Refactor flight control page to use React - @RichardPilbery (tricky_dicky#3571)
```

Comment:
You should put this at the end of the changelog

Reply:
It was when I last updated the changelog...